### ISSUE TITLE
Text Fields

### ISSUE THREAD

**Issue by TheSharpieOne** [xotahal/react-native-material-ui#78]
Any plans to implement text fields? https://material.io/guidelines/components/text-fields.html

I'm pretty new to RN, but have experience with react. I'd be willing to attempt a PR if interested.

**Comment by HofmannZ** [comment]
I agree, I'm using [this](https://github.com/evblurbs/react-native-md-textinput) package right now, but it's no longer actively maintained. We could adapt it to react-native-material-ui.

**Comment by xotahal** [comment]
We don't use those animated inputs in our app. We use inputs like this:
![resultaa](https://cloud.githubusercontent.com/assets/3531955/21943223/98e755f6-d9d0-11e6-9bc1-2877965308bd.gif)
And for them, you can use `ListItem` component easy.

But I agree, we need the animated inputs to this library for sure :)

**Comment by semiautomatix** [comment]
Any movement on this, I'm using https://github.com/evblurbs/react-native-md-textinput at the moment but would prefer a unified component with themes

**Comment by iRoachie** [comment]
@semiautomatix I'm using [react-native-material-kit](https://github.com/xinthink/react-native-material-kit) for text fields

**Comment by semiautomatix** [comment]
@iRoachie, I've used react-native-material-kit but I am trying to avoid anything that requires native support - I'm also building for Windows 10.

**Comment by iRoachie** [comment]
It doesn't need native support, Its js only. The only thing the `react-native link` is used for is the icons

**Comment by semiautomatix** [comment]
Ok, good to know!

**Comment by fernandopasik** [comment]
The last answer is pretty old, but do you guys know which would be the best alternative for textfields now?

**Comment by semiautomatix** [comment]
@fernandopasik I currently either use react-native-material-kit or react-native-material-textfield, if I just need the textfield?

**Comment by fernandopasik** [comment]
@semiautomatix thanks for the suggestion!

**Comment by adrianicv** [comment]
We can develop a textfield inspired in react-native-material-textfield (https://github.com/n4kz/react-native-material-textfield) and adding a dropdown behaviour option inspired in (https://github.com/n4kz/react-native-material-dropdown).
I've tested the first one in Android, iOS and Web and it works really well, so it could be an awsome asset to have in this library. 
I'll try to work in a PR.
